apiserver/allfacades.go

```diff
@@ -146,7 +146,7 @@ func AllFacades() *facade.Registry {
 	reg("Application", 6, application.NewFacadeV6)
 	reg("Application", 7, application.NewFacadeV7)
 	reg("Application", 8, application.NewFacadeV8)
-	reg("Application", 9, application.NewFacadeV9) // ApplicationInfo, generational config.
+	reg("Application", 9, application.NewFacadeV9) // ApplicationInfo, generational config, Force on App and Unit Removal.
```

Comment:
I love it when I need to add something to an API and it's already been version-bumped for me!

Reply:
Me too! Thank you, team :D